docs/general/testing/web/index.md

```diff
@@ -0,0 +1,25 @@
+---
+uid: testing-web-clients
+title: Testing Jellyfin Web Clients
+---
+
+## Weekly Unstable Builds
+
+Weekly unstable builds ship Jellyfin Server and Jellyfin Web in a complete package. Please refer to [Testing Jellyfin Server](/docs/general/testing/server/) for more info.
+
+## Testing PRs from Cloudflare Pages Deployments
+
+Each PR of [Jellyfin Web](https://github.com/jellyfin/jellyfin-web) and [Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue) is automatically deployed to Cloudflare Pages. Please follow the steps below on how to test them.
+
+1. Find the PR you would like to checkout in the list of open PRs: [PRs for Jellyfin Web](https://github.com/jellyfin/jellyfin-web/pulls), [PRs for Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue/pulls)
+2. On most PRs, there should be a comment from `jellyfin-bot` with a link to a deployed version of the current PR. Click on the link next to `Preview URL` to open a hosted instance of the current PR. Some PRs might not have this comment, or the link is otherwise not working. It will have to be built from source in this case.
```

Comment:
>Some PRs might not have this comment, or the link is otherwise not working. It will have to be built from source in this case.

This should probably be a note section/exception (`:::note`?) for the odd case. I'd also word it a bit different to make it feel more natural:

>Some PRs might not have a preview URL comment, or the link might otherwise not work. To test out these PRs, you will need to build them manually.